### DISCUSSION TITLE
stra_exit_long平仓错误

### DISCUSSION THREAD

**Discussion by Guan2019310358** [wondertrader/wtpy#127] [General]
`curPos = context.stra_get_position(code,True) `
`context.stra_exit_long(code, curPos, 'exitlong')`
以上代码平掉所有仓位，无法实现T+1式平仓。

`context.stra_exit_long(code, 300, 'exitlong')`
该方法的平仓数量有时候会出现错误。

**Comment by shaoyucheng** [comment]
用context.stra_set_position(code, 0)吧，不用考虑持仓方向问题。
